File: src/main/java/com/hohltier/decompiler/menu/help/Help.java
package com.hohltier.decompiler.menu.help;

import com.hohltier.decompiler.services.LanguageService;

import javax.swing.*;
import java.awt.*;
import java.awt.event.KeyEvent;

public class Help extends JMenu {

    public Help() {
        setText(LanguageService.getTranslation("help"));
        setMnemonic(KeyEvent.VK_H);

        add(new HelpAbout());

        if (Desktop.isDesktopSupported() && Desktop.getDesktop().isSupported(Desktop.Action.APP_ABOUT))
            setVisible(false);
    }

}

File: src/main/kotlin/dev/shota/decompiler/old/menu/help/Help.kt
package dev.shota.decompiler.old.menu.help

import dev.shota.decompiler.old.menu.Menu
import dev.shota.decompiler.old.menu.help.items.About
import java.awt.Desktop
import java.awt.event.KeyEvent

class Help : Menu("help", KeyEvent.VK_H) {

    init {
        isVisible = Desktop.isDesktopSupported() && !Desktop.getDesktop().isSupported(Desktop.Action.APP_ABOUT)

        add(About())
    }

}